xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -55,34 +64,62 @@ window.MSCheckbox = Class.create({
     ];
     this.labels = ['','',''];
 
+    
+    
+    this.button = document.createElement("button");
+    this.button.className = "rights-edit";
+    this.button.onclick = this.createClickHandler(this);
```

Comment:
This looks like a quite old way to add event listener, wouldn't it be better to use [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)? I don't know what the best practices are, but may feeling is that the current way is not a best practice.